uploader/s3.go

```diff
@@ -90,24 +93,43 @@ func (u *S3Uploader) Upload(ctx context.Context, local string, remote string, ct
 	return u.uploadFile(ctx, f, remote, contentType)
 }
 
+// CountingReader is a wrapper of io.Reader to count number of bytes read
+type CountingReader struct {
```

Comment:
Why is `CountingReader` exposed outside of the package? It should be `countingReader` if it's not used outside of the package (equivalent to private in Java).